src/metabase/db/metadata_queries.clj

```diff
@@ -93,32 +93,47 @@
 
 (def TableRowsSampleOptions
   "Schema for `table-rows-sample` options"
-  (s/maybe {(s/optional-key :truncation-size) s/Int}))
+  (s/maybe {(s/optional-key :truncation-size) s/Int
+            (s/optional-key :rff)             s/Any}))
 
-(s/defn table-rows-sample :- (s/maybe si/TableSample)
-  "Run a basic MBQL query to fetch a sample of rows belonging to a Table. Can pass in optional `:truncation-size` for
-  text fields which will be honored only if the driver supports expressions."
+(defn- table-rows-sample-query
+  "Returns the mbql query to query a table for sample rows"
+  [table fields {:keys [truncation-size] :as _opts}]
```

Comment:
I'm not sure why we had `:as _opts` before but it doesn't look like it's used anywhere. Not a big deal tho

Reply:
it used to just be destructured out but now the query builder is a separate function so the whole opts are passed to that. I'm a fan of `_unused` instead of just an underscore. I think this might go against the grain of the coding style here though.